classList.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Element.classList utility functions
+ * @module delite/classlist
+ * */
+define([], function () {
+
+	var classList = /** @lends module:delite/classList */ {
+		/**
+		 * Toggle one or multiple classes helper method.
+		 *
+		 * @param {Element} node
+		 * @param {String} value
+		 * @param {Boolean} force
+		 */
+		toggleClass: function (node, value, force) {
+			if (value && value.trim() !== "") {
```

Comment:
You don't actually need this `if()` wrapper, because blank values will be handled (or should be handled) by `_getValues()`, `addClass()`, and `removeClass()`